app/javascript/shared/composables/useMessageFormatter.js

```diff
@@ -14,11 +14,12 @@ export const useMessageFormatter = () => {
    * @param {boolean} isAPrivateNote - Whether the message is a private note.
    * @returns {string} - The formatted message.
    */
-  const formatMessage = (message, isATweet, isAPrivateNote) => {
+  const formatMessage = (message, isATweet, isAPrivateNote, linkify) => {
```

Comment:
Adding so many boolean params is not that great... We could convert the function signature to something like this  

```js
const formatMessage = (message, options) => {
   const defaultOptions = { isATweet: false, isAPrivateNote: false, linkify: true }
   // implementation
}
```

Reply:
Resolving this comment as discussed in the call, as we are not doing it now. The reason is that we need to update it almost everywhere. If we add more parameters, we need to update this. Added a TODO comment in the code as a reminder.